fmriprep/config.py

```diff
@@ -157,6 +157,12 @@
     pass
 
 
+def _gen_run_uuid():
+    from uuid import uuid4
+    from time import strftime
+    return f"{strftime('%Y%m%d-%H%M%S')}_{uuid4()}"
+
+
```

Comment:
Let's not pollute with more short-lived utility functions, whenever convenient.
```suggestion
```

Reply:
unless we plan on add functionality to `config.load` to avoid certain fields, the easiest way to do this is to set run_uuid again after loading. it seems sloppy to manage the specific syntax in two places, whereas having a utility function will make maintenance easier in the long run

Reply:
I think adding a `skip` argument to `config.load` could be very useful here (and in the future) - and in this particular case workarounds your legitimate concern.